app/serializers/bookable_slot_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class BookableSlotSerializer < ActiveModel::Serializer
+  attribute :start_at, key: :start
+  attribute :end_at, key: :end
+  attribute :resourceId do
```

Comment:
That's not what I was saying - why use the block style when the others are using the `key`? It's no big deal but thought I'd clarify.